crates/biome_js_analyze/tests/specs/a11y/useSemanticElements/invalid.jsx

```diff
@@ -7,14 +7,14 @@
     <div role="separator" ></div>
     <div role="button" ></div>
     <div role="article" ></div>
-    <div role="dialog" ></div>
+
     <div role="cell" ></div>
     <div role="columnheader" ></div>
-    <div role="definition" ></div>
```

Comment:
We miss related concepts for these two roles. I plan to add them back in a future PR.